### PR TITLE
Add reinforcement recipes

### PR DESCRIPTION
**Pull request by Maxopoly**
@ProgrammerDan If you wanna wanna double check, feel free to do so. If you dont, just merge.

### CODE REVIEW

File: configCivcraft.yml [hunk 1]
```diff
@@ -42,7 +42,54 @@ factories:
     - Upgrade_to_Laboratory
     - Upgrade_to_Wood_Processor
     - Upgrade_to_Basic_Forge
+    - Upgrade_to_Basic_Fortification
     
+  #Reinforcement factories
+  #Basic Contraption --> Basic Fortification
+  basicFortification:
+    type: FCCUPGRADE
+    name: Basic Fortification
```
Comment (ProgrammerDan) [inline configCivcraft.yml]: Yeah lets use the plural form-- Fortifications instead of Fortification.

File: configCivcraft.yml [hunk 2]
```diff
@@ -42,7 +42,54 @@ factories:
...
+    recipes:
+     - Produce_decent_solid_Miner_Component
+     - Produce_decent_shiny_Miner_Component
```
Comment (ProgrammerDan) [inline configCivcraft.yml]: As a courtesy what do you think about including the basic recipes so people can more easily mass produce them once they have the factory online?

Comment (Maxopoly) [inline configCivcraft.yml]: Why not. Might as well make them produce in bigger quantities then, how about doubling everything?

Comment (Maxopoly) [inline configCivcraft.yml]: Or actually the component stuff could just be quadrupled, it's super low anyway

File: configCivcraft.yml [hunk 3]
```diff
@@ -10952,6 +10999,315 @@ recipes:
...
+      strongHunter:
+        material: GHAST_TEAR
+        name: Hunter Component
+        lore: Strong Reinforcement Ingredient
+        amount: 6
+      xp:
+        material: EMERALD_BLOCK
+        amount: 12
+    output:
+      material: INK_SACK
```
Comment (ProgrammerDan) [inline configCivcraft.yml]: Should be fireworks star

File: configCivcraft.yml [hunk 4]
```diff
@@ -10952,6 +10999,315 @@ recipes:
...
+      sand:
+        material: SAND
+        durability: -1
+        amount: 64
+      gravel:
+        material: GRAVEL
+        amount: 64
+    output:
+      bracers:
+        material: FLINT
```
Comment (ProgrammerDan) [inline configCivcraft.yml]: Let's make it a NetherBrick Item here instead.

File: configCivcraft.yml (hunk 2 above)
Comment (ProgrammerDan) [inline configCivcraft.yml]: Nah, just leave the production time low. So they can turn it on and it'll burn through fuel and produce it just a bit faster then the crafting grid.

**Comment by Maxopoly** [comment]
Ok, that should adress everything

File: configCivcraft.yml [hunk 5]
```diff
@@ -10952,6 +11017,440 @@ recipes:
...
+        durability: 1
+      coalore:
+        material: COAL_ORE
+        amount: 24
+    output:
+      basicMiner:
+        material: FLINT
+        name: Miner Component
+        lore: Basic Reinforcement Ingredient
+        amount: 8
```
Comment (ProgrammerDan) [inline configCivcraft.yml]: Should only produce 1, yeah? Same for other Basics. Otherwise this recipe is not only faster but 8x more efficient.

**Comment by Maxopoly** [comment]
ok NOW we got it :D

File: configCivcraft.yml (hunk 5 above)
Comment (ProgrammerDan) [inline configCivcraft.yml]: This doesn't appear addressed yet?

File: configCivcraft.yml [hunk 6]
```diff
@@ -10952,6 +11011,497 @@ recipes:
...
+      basicRein:
+        material: CLAY_BALL
+        amount: 64
+        lore:
+         - Basic Reinforcement
+         
+  Produce_basic_hunter_reinforcement:
+    type: PRODUCTION
+    production_time: 2s
+    amount: 4
```
Comment (ProgrammerDan) [inline configCivcraft.yml]: This is on the wrong line

**Comment by Maxopoly** [comment]
Ok, fixed that and added a basic hunter and basic miner recipe to the basic contraption

**Comment by ProgrammerDan** [comment]
Looks good, any other problems we won't find until we deploy it.

Let's merge this and then I'll work on a repair cost PR

**Pull request merged**